BizHawk.Client.EmuHawk/tools/TAStudio/PlaybackBox.Designer.cs

```diff
@@ -192,7 +192,7 @@ private void InitializeComponent()
 		private System.Windows.Forms.CheckBox AutoRestoreCheckbox;
 		private System.Windows.Forms.CheckBox TurboSeekCheckbox;
 		private System.Windows.Forms.CheckBox FollowCursorCheckbox;
-		private System.Windows.Forms.CheckBox RecordingModeCheckbox;
+		public System.Windows.Forms.CheckBox RecordingModeCheckbox;

```

Comment:
You can't do this.  This is auto-generated code.  As soon as a change is made, it will put it back to private again.

Instead, if you need access to a proprety of one of these controls make a public property getter/setter in the non-generated part of this class:
public bool RecordingIsChecked { get { return RecordingModeCheck.Checked; } { set { RecordingModeCheck.Checked = value; } }